test/unit/testWillCodicils.js

```diff
@@ -15,23 +15,37 @@ describe('WillCodicils', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
```

Comment:
Please update the description for this test as it is for will codicils but it is referring to deceased address.